build-and-deploy-docker-image-aks.yml

```diff
@@ -0,0 +1,51 @@
+on: [push]
```

Comment:
Change the name of the file from build-and-deploy-docker-image-aks.yml  to build-and-deploy-docker-image-to-aks-using-manifest.yml